Document metrics temporality for time series data streams (#7174)
elastic/docs-content: manage-data/data-store/data-streams/downsampling-concepts.md

  * `gauge` fields:
    * Stores `min`, `max`, `sum`, and `value_count` as `aggregate_metric_double`.
  * `counter` fields:

>     * For [cumulative](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) counters: stores the first value, and stores up to two data points if a counter reset is detected {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga`. The field type is preserved. In previous versions, only the last value is stored.

Reviewer: ```suggestion
    * For [cumulative](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) counters: stores the first value, and stores up to two data points if a counter reset is detected {applies_to}`stack: ga 9.4`. The field type is preserved. In previous versions, only the last value is stored.
```

for inline tags, serverless:ga is implicit

we only need to spell out serverless: ga in page-level metadata (in frontmatter)

>     * {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` The metric temporality is preserved. This means that for delta counters, downsampling stores the sum of values in the bucket.

Reviewer 2: ```suggestion
    * {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` For delta counters: stores the sum of values in the bucket.
```

Reviewer 2: And maybe link to a page explaining cumulative and delta.

>     * {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` For [delta](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) counters: stores the sum of values in the bucket.

Reviewer: ```suggestion
    * {applies_to}`stack: ga 9.5` For [delta](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) counters: stores the sum of values in the bucket.
```

  * `histogram` fields {applies_to}`stack: preview 9.3` {applies_to}`serverless: preview`
    * Merges individual histograms into a single histogram, preserving the field type. 
    
      All three histogram field types are supported: [`histogram`](elasticsearch://reference/elasticsearch/mapping-reference/histogram.md), [`tdigest`](elasticsearch://reference/elasticsearch/mapping-reference/t-digest.md), and [`exponential_histogram`](elasticsearch://reference/elasticsearch/mapping-reference/exponential-histogram.md). 
      
      The `histogram` and the `tdigest` field type use the [T-Digest](elasticsearch://reference/aggregations/search-aggregations-metrics-percentile-aggregation.md) algorithm for merging.

>     * {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` The metric temporality is preserved. For [cumulative](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) histograms, we use the same algorithm as for cumulative counters.

Reviewer: ```suggestion
    * {applies_to}`stack: ga 9.5` The metric temporality is preserved. For [cumulative](/manage-data/data-store/data-streams/metric-temporality.md#cumulative-vs-delta) histograms, {{es}} use the same algorithm as for cumulative counters.
```
